jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/wih/AsyncWorkItemHandler.java

```diff
@@ -29,6 +29,7 @@
 import org.kie.api.runtime.process.WorkItemHandler;
 import org.kie.api.runtime.process.WorkItemManager;
 import org.kie.api.runtime.query.QueryContext;
+import org.kie.internal.runtime.Cacheable;
```

Comment:
Just curious, does it provide any benefits to have WIH cacheable? Maybe performance improvement?

Reply:
it means that the work item handler does not have to be recreated for each execution - especially important for process instance and request strategies.